src/webgpu/api/validation/encoding/queries/resolveQuerySet.spec.ts

```diff
@@ -1,13 +1,94 @@
 export const description = `
-TODO:
-- invalid GPUQuerySet
```

Comment:
I don't see any tests with _invalid_ `GPUQuerySet`s, i.e. attempting to use `GPUQuerySet`s that failed during creation.

Can you either leave a TODO (or better, a `.unimplemented()` test) for that?